t/33-vagrant.t

```diff
@@ -0,0 +1,511 @@
+#!/usr/bin/perl
+
+use Cwd qw(abs_path);
+use Test::Most;
+use File::chdir;
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '20';
```

Comment:
for me gladly the test module only takes a second to execute so I suggest to lower this. We use 5 as the lowest value in most test modules.